src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const styleObject = {};
+  const styleCommands = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy of the array before operating on it. The `split` method does not mutate the original string, so you can chain other methods directly without assigning it to `styleCommands` first.